GOVERNANCE.md

```diff
@@ -0,0 +1,235 @@
+private. This is to allow Code Owners to express their opinions about a nominee without causing embarrassment freely.
+The approval requires majority vote (>50%) from the Code Owners.
+
+#### Earning a Nomination
+
+The nomination is at the discretion of the existing Code Owners and there is no single path of earning a nomination for,
+however, we can give some guidance about some actions that would help:
+
+*   Start by expressing interest to the Code Owners that you are interested in becoming a Code Owner.
+*   You can start tackling issues labeled as ‘help wanted’, or if you are new to the project, some of the ‘good first issue’ tickets.
+*   As you gain experience with the codebase and our standards, we will ask you to do code reviews for incoming PRs (i.e., all Code Owners are expected to shoulder a proportional share of community reviews).
+*   We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing Code Owners.
+
+#### Losing Code Owner Status
+
+If a Code Owner is no longer interested and cannot perform the Code Owner duties listed above, they can volunteer to be
+moved to emeritus status. The Code Owner status is attributed for life otherwise. An emeritus Code Owner may request
+reinstatement of commit access from the rest of Code Owners. Such reinstatement is subject to lazy consensus approval of
+active Code Owners.
+
+Emeritus status is a nominal title, and confers no special rights (like voting) or access. Emeritus members are
+functionally identical to normal contributors, with the exception that they can request for reinstatement of their
+commit access.
+
+In extreme cases, Code Owners can lose their status by a vote of the Code Owners per the voting process below.
+
+## Decision Making Process
+
+Decisions about the future of Llama Stack are made through discussion with all community members, from the newest user
+to the most experienced maintainer. All non-sensitive project management discussion takes place on the project issue
+tracker system. Occasionally, sensitive discussion occurs on a private channel of our Discord.
+
+To ensure that the project is not bogged down by endless discussion and continual voting, the project operates a policy
+of lazy consensus. This allows the majority of decisions to be made without resorting to a formal vote.
+
+
+### Lazy consensus
+
+Decision making typically involves the following steps:
+*   Discussion *(in Discord channels and GitHub)*
+*   Proposal *(via GitHub issue + GitHub PR)*
+*   (optional) Maintainers voting (if there is active opposition + consensus is not reached through discussion)
+*   Decision
+
+Any community member can make a proposal for consideration by the community. To initiate a discussion about a new idea,
+they should create an issue or submit a PR implementing the idea to the issue tracker. This will prompt a review and,
```

Comment:
nit: what is "the issue tracker"? what does it mean to submit a PR to it?

Reply:
removed.